app/graphql/types/payloads/sign_in_payload.rb

```diff
@@ -1,6 +1,7 @@
 module Types
   module Payloads
     class SignInPayload < Types::BaseObject
+      field :current_user, CurrentUserType, null: false, method: :user
```

Comment:
Why `null: false`?

Reply:
it's my mistake. Yes, when user is nil-object, field(current_user) must be null